corehq/apps/es/management/commands/restore_es_snapshot.py

```diff
@@ -88,6 +88,6 @@ def rewind_pillows(date):
                                                                     date_updated=date)
                 seq = checkpoint.seq
             except HistoricalPillowCheckpoint.DoesNotExist:
-                seq = DEFAULT_EMPTY_CHECKPOINT_SEQUENCE
+                seq = DEFAULT_EMPTY_CHECKPOINT_SEQUENCE[checkpoint.sequence_format]
```

Comment:
i don't think this works because the checkpoint model in this case is a  `HistoricalPillowCheckpoint`. though i do think you can use `pillow.checkpoint.sequence_format`